### PR TITLE
Rs 2023.02.04

### PR DESCRIPTION
**Pull request by rschlaikjer**
Literally every page here is packed to the gills why you gotta do us like this

2023-02-04-185510_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 13/13_01_02_CIEL13_7D.txt [hunk 1]
```diff
@@ -111,82 +119,99 @@ Both her physical and mental capabilities were as average as it gets, so it's on
...
+ She had a permanently tormented expression,
+ and I honestly half-expected her to retire.
+ She'd always perk right up after airing out her frustrations with me, though.
+ I was honestly amazed."
 }
 [sha:67048d892f9f0d34ac529753832dca6777e20b06]{
 -- Page 356, Offset 36323.
 -- 「それから２年たって、わたしたちはチームになりました。
-"We became a team two years after that.
+"It was over two years later that we became a team.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:151]: Hmm, I prefer the OG TL, but probably more in this form. Simple and nice.
>Two years passed, and we became a team.

File: script/Ciel/Day 13/13_01_02_CIEL13_7D.txt [hunk 2]
```diff
@@ -111,82 +119,99 @@ Both her physical and mental capabilities were as average as it gets, so it's on
...
 }
 [sha:ab05503611014366fecbd58d967ea95af62035cd]{
 -- Page 356, Offset 36324.
 -- ちょっとした失敗で司祭さまの不興を買った彼女が、泣く泣くわたしを頼って来たんです。自分を弟子にしてほしい、更迭から守ってほしいって。彼女からすれば死ぬほどイヤな選択でしょうけど、生き残るにはそれしかなかった。
-She made a small mistake that upset the Presbyter, so she ended up coming to me in tears. She begged me to make her my apprentice and to keep her from being dismissed. She probably hated it to death, but that was the only way she could survive.
+She'd made some small mistake that invoked the ire of the Presbyter, and came crying to me for help.
+ She begged me to take her on as my apprentice and keep her from being dismissed.
+ I'm sure she hated it to death, but it was the only way she could survive.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:158]: It was just some random 司祭さま so I'd word it more like this.
>She'd made a small mistake that invoked the ire of some Presbyter, and...

File: script/Ciel/Day 13/13_01_02_CIEL13_7D.txt [hunk 3]
```diff
@@ -111,82 +119,99 @@ Both her physical and mental capabilities were as average as it gets, so it's on
...
-I was scouted by the Burial Agency, so I had a certain amount of authority. I knew that she would be a burden, but I took her on as my apprentice regardless.
+I'd already been scouted by the Burial Agency, so I had a certain amount of authority.
+ Even though I knew she'd be a burden, I took her on as my apprentice.
 }
 [sha:b13daceeffe0a431882bd0c4fc06fcfce3fa0e9d]{
 -- Page 356, Offset 36326.
 -- それからずっと、任務以外では口をきかない関係のまま、５年間が過ぎました」
-From then on, we never spoke to each other aside from whenever we were out on missions. We maintained that relationship for five years."
+After that, we never spoke to each other except during missions.
+ We maintained that relationship for five years."
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:170]: The それからずっと takes the 関係 so it's weird for them to be seperated.
Here's an alternative that's more how I think about it.
>After that, we kept our relationship distant. For five whole years, we never spoke to each other except during missions.

File: script/Ciel/Day 13/13_01_02_CIEL13_7D.txt [hunk 4]
```diff
@@ -111,82 +119,99 @@ Both her physical and mental capabilities were as average as it gets, so it's on
...
 }
 [sha:f5d47481ff059eb95de8b1e02d0c8d1485ee1671]{
 -- Page 357, Offset 36327.
 -- 「……先輩」
-"...Senpai."
+"Senpai."
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:175]: Probably needs the ellipsis
>Senpai...

File: script/Ciel/Day 13/13_01_02_CIEL13_7D.txt [hunk 5]
```diff
@@ -111,82 +119,99 @@ Both her physical and mental capabilities were as average as it gets, so it's on
...
 -- Page 358, Offset 36331.
 -- 息を呑む音がした。
-I hear her gasp.
+She gasps.
 }
 [sha:0218bbd1713a899f8175297a659579a0b06695b9]{
 -- Page 358, Offset 36332.
 -- 淋しげな背中が少しだけ震えたように見えた。
-It seemed like her lonely back shivered a little.
+I feel like I saw her lonely back tremble slightly.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:203]: Believe it or not 寂しげな背中 is a collocation/set phrase in Japanese. Probably shouldn't be literal.
Not entirely sure how to go about it, but something like this.
>I feel like I just saw her forlorn figure tremble slightly.

File: script/Ciel/Day 13/13_01_02_CIEL13_7D.txt [hunk 6]
```diff
@@ -111,82 +119,99 @@ Both her physical and mental capabilities were as average as it gets, so it's on
...
+ Face still hidden, she fights to suppress the emotions welling up inside of her.
 }
 [offset:36334]{
 -- Page 359, Offset 36334.
 -- 「……ごめん、なさい」
-"...I'm... sorry."
+"I'm... sorry."
 }
 [sha:d6f9be0f7b5ee917d31fbccab807280dc1ff83d0]{
 -- Page 359, Offset 36335.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:217]: 36336
Elegance required.
The Japanese structure is very much useable.
ただ is basically emphatic here so no need to keep it too close.
>Silently, without so much as a tear, she continues to weep.

Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:217]: Current ver reads more elegantly to me. "Without so much as a tear" comes off as quite unnatural.

Comment (Petrikow) [reply script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:217]: ITS BORING!!!

Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:217]: Sorry but I'm a strong disagree on this one. We can bring it up again at GPR if you want.

Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:217]: Could compromise to this to keep the structure? 
> Silently, without shedding a single tear, she continues to weep.

Comment (Petrikow) [reply script/Ciel/Day 13/13_01_02_CIEL13_7D.txt:217]: Sure

**Pull request merged**